requirements/dev.txt

```diff
@@ -0,0 +1,235 @@
+#
+# This file is autogenerated by pip-compile with python 3.6
+# To update, run:
+#
+#    pip-compile --output-file=requirements/dev.txt requirements/dev.in
+#
+-e .
+    # via -r requirements/dev.in
+-e git+https://github.com/kytos-ng/kytos.git#egg=kytos
+    # via -r requirements/dev.in
+-e git+https://github.com/kytos-ng/of_core.git#egg=kytos_of_core
+    # via -r requirements/dev.in
+-e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
+    # via
+    #   -r requirements/dev.in
+    #   kytos
+-e git+https://github.com/amlight/flow_stats@fix/install_req_deps#egg=amlight-flow_stats
```

Comment:
Once [`flow_stats` PR 12](https://github.com/amlight/flow_stats/pull/12) lands I'll revert this temporary branch `fix/install_req_deps` usage here, it turns out without this, it would bring unwanted dependencies here.